ga4gh/datamodel/variants.py

```diff
@@ -437,13 +485,14 @@ def convertVariant(self, record, callSetIds):
         sampleData = record.__str__().split()[9:]  # REMOVAL
         variant.calls = []
         sampleIterator = 0  # REMOVAL
-        for name, call in record.samples.iteritems():
-            if self.getCallSetId(name) in callSetIds:
-                genotypeData = sampleData[sampleIterator].split(
-                    ":")[0]  # REMOVAL
-                variant.calls.append(self._convertGaCall(
-                    record.id, name, call, genotypeData))  # REPLACE
-            sampleIterator += 1  # REMOVAL
+        if callSetIds:
```

Comment:
Why are we changing this here? Shouldn't `record.samples.iteritems` be empty if no samples are present?